bottlerocket-settings-models/modeled-types/src/kubernetes.rs

```diff
@@ -1440,3 +1441,75 @@ mod test_hostname_override_source {
         }
     }
 }
+
+// =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
+// Define the DevicePlugin struct
+#[model(impl_default = true)]
+pub struct K8sDevicePluginsSettings {
+    nvidia: NvidiaDevicePluginSettings,
+}
+
+/// NvidiaRuntimeSettings contains the container runtime settings for Nvidia gpu.
+#[model(impl_default = true)]
+pub struct NvidiaDevicePluginSettings {
+    pass_device_specs: bool,
+    device_id_strategy: NvidiaDeviceIdStrategy,
+    device_list_strategy: NvidiaDeviceListStrategy,
+}
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum NvidiaDeviceIdStrategy {
+    Uuid,
+    Index,
+}
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum NvidiaDeviceListStrategy {
+    Envvar,
+    VolumeMounts,
```

Comment:
Does `device-list-strategy = "volume-mounts"` here imply that `visible-devices-as-volume-mounts` should also be `true`?